backend/app/assets/javascripts/spree/backend/format_money.js.erb

```diff
@@ -0,0 +1,27 @@
+//= require spree/backend/translation
+//= require solidus_admin/accounting
+
+Spree.currencyInfo = <%=
+Money::Currency.all.map { |c|
+  format =
+    if c.symbol == "" || c.symbol_first
+      "%s%v"
+    else
+      "%v %s"
+    end
+  [c.id.to_s.upcase, [
+    c.symbol || "¤",
+    c.exponent,
+    format
+  ]]
+}.to_h.to_json
+%>;
```

Comment:
A fair point, I hadn't considered them changing often enough to warrant this, but I suppose if they're changed at all the existing method would be a pain.

I've pushed up a change which combines this and the existing translations into a `_js_locale_data` partial